.travis/gcc-latest/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM gcc:9.2.0
```

Comment:
@bourtemb Because this then suddently results in failing builds when a potentially new gcc version is released. And I don't want to get into the habit of accepting PRs with failed builds.

With having an explicit version, we can upgrade the versions when we have time.

Reply:
> Having a similar llvm/clang container would be useful indeed.

Okay so I'll make another PR.

Reply:
> @bourtemb Because this then suddently results in failing builds when a potentially new gcc version is released. And I don't want to get into the habit of accepting PRs with failed builds.

Makes sense.

> With having an explicit version, we can upgrade the versions when we have time.

Sure. We need to keep an eye on new gcc releases regularly.

Reply:
Shouldn't be this gcc:latest or smth similar?

Reply:
> Shouldn't be this gcc:latest or smth similar?

@Ingvord, I asked the same question yesterday. The conversation was marked as resolved, this is probably why you didn't see it.
Here is the reply from @t-b : 

> @bourtemb Because this then suddently results in failing builds when a potentially new gcc version is released. And I don't want to get into the habit of accepting PRs with failed builds.
> 
> With having an explicit version, we can upgrade the versions when we have time.

Reply:
OK, then probably we should rename the folder to let's say just `gcc`